microsoft-edge/progressive-web-apps-chromium/how-to/digital-goods-api.md

```diff
@@ -35,17 +35,50 @@ The Payment Request API⁠⁠ handles the actual payment transaction when a purc
 See [Payment Request API](https://developer.mozilla.org/docs/Web/API/Payment_Request_API) at MDN.
 
 
+<!-- ====================================================================== -->
+## Enable the Digital Goods API
+
+The Digital Goods API is currently available in pre-release versions of Microsoft Edge.  To use the API, either enable the feature locally in Microsoft Edge, or register in the Digital Goods API origin trial to test the API in production.
```

Comment:
is it possible to test in production?  how do users get support in their Edge?

Reply:
Could you clarify what you mean by 'test in production' in this context? Thanks.

Reply:
"Test in production" means enabling the API in your production application, so that real users of your app can use the features of your app that depends on the API.